src/main/java/com/cisco/trex/util/TrexServerMode.java

```diff
@@ -0,0 +1,21 @@
+package com.cisco.trex.util;
+
+/**
+ * Class represent for Trex Server Mode
+ */
+public enum TrexServerMode {
```

Comment:
first two letters should be capital
"TRexServerMode"

Reply:
done